x-pack/qa/openldap-tests/src/test/java/org/elasticsearch/xpack/security/authc/ldap/SearchGroupsResolverTests.java

```diff
@@ -202,7 +212,7 @@ public void testReadBinaryUserAttribute() throws Exception {
 
     @Override
     protected String ldapUrl() {
```

Comment:
This is currently not used anymore because of the way `GroupsResolverTestCase` is setup. We need to cleanup GroupsResolverTestCase to not rely on class path resources that are outside of a jar. it shouldn't matter. But this rework is out of scope of this PR.